src/frontend/shared-components/.storybook/main.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ["../src/**/*.stories.js"],
+  stories: ["../src/**/**/*.stories.js"],
```

Comment:
Was the original not catching it? /**/ should go down any number of nested dirs, /**/**/ should be redundant?